Gemfile.lock

```diff
@@ -330,8 +330,8 @@ GEM
       thor (~> 0.19.1)
     formatador (0.2.5)
     gli (2.15.0)
-    globalid (0.4.0)
```

Comment:
not intentional, they're a result of an oversight when merging. It shouldn't make a difference but let's go with the later version for these couple of gems.